recipe/meta.yaml

```diff
@@ -7,12 +7,13 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/affinegap-{{ version }}.tar.gz
-  sha256: 02faa7579df8d98beafd40bb924b7a3a9d4e42edf6938e297366903054e4ef61
+  # Use upstream archive tarball for tests
+  url: https://github.com/dedupeio/affinegap/archive/refs/tags/v1.12.tar.gz
```

Comment:
```suggestion
  url: https://github.com/dedupeio/affinegap/archive/refs/tags/v{{ version}}.tar.gz
```

Reply:
Hmm, I need to improve my code generator...